tests/integration/testdata/package/aws-serverlessrepo-application.yaml

```diff
@@ -12,5 +12,5 @@ Metadata:
     ReadmeUrl: ./README.md
     Labels: ['tests']
     HomePageUrl: https://github.com/user1/my-app-project
-    SemanticVersion: 0.0.1
+    SemanticVersion: 1.0.1
```

Comment:
Why shouldn't we leave this as `0.0.1`?

Reply:
this is failing because our output is '0.0.1' and awscli's output is 0.0.1 which is failing. Thus I changed the leading 0 to 1 so we don't wrap it in quotes.